arangod/Aql/IResearchViewExecutor.h

```diff
@@ -260,7 +264,7 @@ class IResearchViewExecutorBase {
     // .
     std::vector<ValueType> _keyBuffer;
     std::vector<AqlValue> _scoreBuffer;
-    std::vector<irs::bytes_ref> _sortValueBuffer;
+    std::vector<std::unordered_map<int, irs::bytes_ref>> _storedValueBuffer;
```

Comment:
can't we use something like `std::vector<std::vector<irs::bytes_ref>>` instead to avoid hash map lookups?